src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml

```
@@ -1,20 +1,30 @@
 twig_hooks:
     hooks:
-        'sylius_shop.account.address_book.create':
+        'sylius_shop.account.address_book.create.layout':
```

I'm not convinced to the `layout` name 🤔 
```suggestion
        'sylius_shop.account.address_book.create.content':
```